tests/unit/views/HelloWorld.test.js

```diff
@@ -1,7 +1,10 @@
 import testHelpers from '@@/unit/testHelpers.js';
-
 import HelloWorld from '@/views/HelloWorld.vue';
 
+function normalizeAmp(html) {
+  return html.replace(/&amp;/g, '&');
+}
+
```

Comment:
Don't do this. DOM serialization is handled at the Snapshot Serializer level. HappyDOM is now more accurately reporting `&lt;` instead if `<`, `&gt;` instead of `>` and `&amp;` instead of `&`.

The snapshots should be updated to now match this more accurate reporting from HappyDOM.

Reply:
Done, removed the normalization and updated the snapshot with only the line which caused the issue changed